### ISSUE TITLE
Add Material Theme UI

### ISSUE THREAD

**Issue by bachig26** [kannagi0303/yt-dlp-gui#34]
I want to request for adding the material theme UI especially Monokai Theme.

Like a selector option to change themes in the settings or from the advanced.

**Comment by kannagi0303** [comment]
Hi, @bachig26 

I'm not sure how to respond to your request.
Microsoft's WPF framework can be highly customized, but the definition file is quite large.
When I start developed it, I didn't think about using other themes.
Implementing the switch theme feature requires a lot of care and time.
Implementing it with the current code may cause errors in some themes.

Since the GUI is just a simple tool, I'm not sure it's worth implementing this feature.
And most importantly, as far as I know, the users of `yt-dlp-gui` are quite few at present (Version `2022.10.10` only has about 360+ downloads)

**Comment by ShiSheng233** [comment]
If so, you might consider using <https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit>.
Although I recommend you to try [WinUI3](https://learn.microsoft.com/en-us/windows/apps/winui/winui3/), which is more in harmony with Windows UI.

**Comment by kannagi0303** [comment]
I have reviewed the excellent `MaterialDesignInXamlToolkit`, but opted for a custom look for a more compact interface.
Although it can be modified to meet my expectations,
However, it is much lighter and simpler to modify from scratch.

Regarding `WinUI3 (MAUI)`, I am also very interested in this. I have tried to create a project in the past, but the project created must contain the App ID, and a single executable file cannot be created separately.
It is currently unknown if there is already support to build it.

If there is any relevant update information, please let me know.
In conclusion, if there is a better solution, I prefer to replace it more aggressively :)
